marconi-chain-index/src/Marconi/ChainIndex/Experimental/Indexers/SyncHelper.hs

```diff
@@ -52,20 +51,23 @@ syncInsertPlan :: Core.InsertPointQuery
 syncInsertPlan =
   Core.InsertPointQuery [sql|INSERT INTO sync (slotNo, blockHeaderHash) VALUES (?, ?)|]
 
+-- TODO Revisit comment
+
 -- | Query for the last point of an indexer
-syncLastPointQuery :: Core.GetLastSyncQuery
-syncLastPointQuery = Core.GetLastSyncQuery $ syncHistoryQuery 1
+syncLastPointsQuery :: Core.GetLastSyncPointsQuery
+syncLastPointsQuery = Core.GetLastSyncPointsQuery syncHistoryQuery ":limit"
+
```

Comment:
See my comment about GetLastSyncPointsQuery in the `SQLiteIndexer` the `":limit"` is probably useless here.

Reply:
What comment are you referring to?

Reply:
I may have forgot to validate it. I'd prefer to have a `newtype GetLastSyncPointsQuery = GetLastSyncPointsQuery (Word -> SQL.Query)` to improve type-safety. I know that SQLite isn't type safe and we can't do much about it, but adding an unsafety layer by relying on parameter names encoded as text makes it worse.
That's also why I prefer the existing implementation of `syncHistoryQuery` (In this module) rather than the one proposed in this PR.

Reply:
Yeah you're right. Changing it.